ddocs/medic-client/temp

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Leaving a note here to remind whoever merges this to delete this file once this is merged: https://github.com/medic/medic/pull/5640